module1/homework/grayscaleImages/main.cpp

```diff
@@ -1,9 +1,10 @@
 #include <array>
 #include <forward_list>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
You could remove array include here since it's in the header.

Reply:
+1